tox.ini

```diff
@@ -26,3 +26,10 @@ commands =
     !integration: pytest --junitxml=.tox/junit-{envname}.xml test/unit
 # Run integration test if in intgration env
     integration: ansible-playbook {posargs}
+
+[flake8]
+# These are things that the devs don't agree make the code more readable
+# E402 module level import not at top of file
+ignore = E402
+# not all the devs believe in 80 column line length
+max-line-length = 160
```

Comment:
Personally, I would prefer to narrow down the lines as 160 sounds too much for me. What's about wrapping lines at 120 columns?

Reply:
in progress